poleno/attachments/models.py

```diff
@@ -110,7 +112,13 @@ def save(self, *args, **kwargs):
             self.size = self.file.size
             self.content_type = magic.from_buffer(self.file.read(), mime=True)
 
-        super(Attachment, self).save(*args, **kwargs)
+            base, extension = os.path.splitext(self.name)
+            base = ''.join([c for c in base if ord(c) < 32][:200])
+            if mimetypes.guess_type(self.name)[0] != self.content_type:
+                extension = guess_extension(self.content_type)
+            self.name = (base or u'attachment') + extension
```

Comment:
Vyborne. Vsetko je super. Len pls, kedze sa nam logika okolo sanitizovania nazvu suboru skomplikovala, vydel ju do `sanitize_filename(filename, content_type)` do `poleno.utils.misc` za `guess_extension`. Nech ju mozeme reusnut na oboch miestach podobne ako reusujeme `guess_extension`. Ku `sanitize_filename` napis do docstringu popis, co funkcia robi a zopar examplov. Vid ako je to v `guess_extension`.

Reply:
oplati sa pridat do funckie parameter default_base, ktory by sluzil ako nazov subora v pripade ze by po odfiltrovani zakazanych znakov ostal prazdny retazec?

def sanitize_filename(filename, content_type, default_base=u'attachment'):
...
 self.name = (base or default_base) + extension

Reply:
a som si nasiel chybu, ma byt [c for c in base if not ord(c) < 32]

Reply:
> oplati sa pridat do funckie parameter default_base

Ano, mozes, len mu daj default hodnotu `default_base=u'attachment'`